docs/source/description.rst

```diff
@@ -6,20 +6,39 @@ ndx-ophys-devices Extension for NWB
-
-- **DeviceModel** extends ``Device`` to hold metadata on the model of the device.
-- **Indicator** extends ``NWBContainer`` to hold metadata on the fluorescent indicator (e.g., label=GCaMP6).
-- **Effector** extends ``NWBContainer`` to hold metadata on the effector/opsin (e.g., label=hChR2).
-- **OpticalFiber** extends ``DeviceModel`` to hold metadata on the optical fiber (e.g., numerical_aperture=0.39).
-- **ExcitationSource** extends ``DeviceModel`` to hold metadata on the excitation source (e.g., excitation_wavelength_in_nm=470.0).
-- **PulsedExcitationSource** extends ``ExcitationSource`` to hold metadata on the pulsed excitation source (e.g., pulse_rate_in_Hz=1000.0).
-- **Photodetector** extends ``DeviceModel`` to hold metadata on the photodetector (e.g., detected_wavelength_in_nm=520.0).
-- **DichroicMirror** extends ``DeviceModel`` to hold metadata on the dichroic mirror (e.g., cut_on_wavelength_in_nm=470.0).
-- **OpticalFilter** extends ``DeviceModel`` to hold metadata on a general optical filter (e.g., filter_type='Bandpass').
-- **BandOpticalFilter** extends ``OpticalFilter`` to hold metadata on any bandpass or bandstop optical filters (e.g., center_wavelength_in_nm=505.0).
-- **EdgeOpticalFilter** extends ``OpticalFilter`` to hold metadata on any edge optical filters (e.g., cut_wavelength_in_nm=585.0).
-- **ObjectiveLens** extends ``DeviceModel`` to hold metadata on the objective lens (e.g., magnification=60.0).
+This extension consists of neurodata types in the following categories:
+
+**Container Classes:**
+
+- **Indicator** extends ``NWBContainer`` to hold metadata on the fluorescent indicator.
+- **Effector** extends ``NWBContainer`` to hold metadata on the effector/opsin.
+- **LensPositioning** extends ``NWBContainer`` to hold metadata on the positioning of a lens relative to the brain.
+- **FiberInsertion** extends ``NWBContainer`` to hold metadata on the insertion of a fiber into the brain.
+
+**Model Classes:**
+
+- **DeviceModel** extends ``Device`` to hold metadata on device models.
+- **OpticalFiberModel** extends ``DeviceModel`` to hold metadata on the optical fiber model.
+- **ExcitationSourceModel** extends ``DeviceModel`` to hold metadata on the excitation source model.
+- **PhotodetectorModel** extends ``DeviceModel`` to hold metadata on the photodetector model.
+- **DichroicMirrorModel** extends ``DeviceModel`` to hold metadata on the dichroic mirror model.
+- **OpticalFilterModel** extends ``DeviceModel`` to hold metadata on a general optical filter model.
+- **BandOpticalFilterModel** extends ``OpticalFilterModel`` to hold metadata on any bandpass or bandstop optical filter models.
+- **EdgeOpticalFilterModel** extends ``OpticalFilterModel`` to hold metadata on any edge optical filter models.
+- **OpticalLensModel** extends ``DeviceModel`` to hold metadata on the optical lens model.
+
+**Device Instance Classes:**
+
+- **DeviceInstance** extends ``Device`` to hold metadata on device instances.
```

Comment:
The proposed change to the NWB schema does not include a `DeviceInstance` type. `Device` is just redefined to be instances or models. Do you think it would be useful to create a new `DeviceInstance` type? I can see some value to say that if we redefine `Device` to be instances, then the inheritance of a `DeviceModel` from a `Device` doesn't quite make sense, but it is simpler. Could you add your thoughts on this to https://github.com/NeurodataWithoutBorders/nwb-schema/pull/608 ?